<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingUp</class>
 <widget class="QDialog" name="SingUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
  background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(39, 81, 229, 255), stop:1 rgba(216, 177, 255, 255));
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>641</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
    background:#d9faff;
	border-radius:25px;
    border-bottom-left-radius: 25px;
	border-top-left-radius: 25px;
    box-shadow: -20px -32px 72px  rgb(0, 0, 0, 50),
                2px 2px 5px rgb(94, 104, 121, 50)
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="bottom_welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#bottom_welcome{
    background:#00204a;
	border-radius:0px;
    border-bottom-left-radius: 25px;
border-bottom-right-radius: 25px;
	border-top-left-radius: 0px;
    box-shadow: -20px -32px 72px  rgb(0, 0, 0, 50),
                2px 2px 5px rgb(94, 104, 121, 50)
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Crate_button">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>20</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kalameh</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#Crate_button{
background: #005792;
border-radius:25px;
color:#d9faff;
}
QPushButton#Crate_button:hover{
background: #00bbf0;
border-radius:25px;
color:#d9faff;

}</string>
     </property>
     <property name="text">
      <string>Crete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Singup_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-20</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Kalameh ExtraBold</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#Singup_label{
color: #00204a;

}
</string>
    </property>
    <property name="text">
     <string>Singup</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_logo_s">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>100</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#user_logo_s{
background:rgb(0,0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/D:/grocery_store/grocery_store/assets/icons/user (3).png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_s">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#username_s{
background: #00bbf0;
border-radius:25px;
color:#00204a;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="username_F_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#username_F_s{
background: #00bbf0;
border-radius:25px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="username_befor_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>89</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#username_befor_s{
background: rgb(0,0,0,0);
border-radius:25px;
color:#005792;
}</string>
    </property>
    <property name="text">
     <string>      username</string>
    </property>
   </widget>
   <widget class="QFrame" name="password_F_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#password_F_s{
background: #00bbf0;
border-radius:25px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="password_logo_s">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>230</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#password_logo_s{
background:rgb(0,0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/D:/grocery_store/grocery_store/assets/icons/lock (1).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="password_befor_s">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>219</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#password_befor_s{
background: rgb(0,0,0,0);
border-radius:25px;
color:#005792;
}</string>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_s">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#password_s{
background: #00bbf0;
border-radius:25px;
color:#00204a;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="address_F_s">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>130</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#address_F_s{
background: #00bbf0;
border-radius:25px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="address_logo_s">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#address_logo_s{
background:rgb(0,0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/D:/grocery_store/grocery_store/assets/icons/map.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="address_befor_s">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>89</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#address_befor_s{
background: rgb(0,0,0,0);
border-radius:25px;
color:#005792;
}</string>
    </property>
    <property name="text">
     <string>address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_s">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>130</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#address_s{
background: #00bbf0;
border-radius:25px;
color:#00204a;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="phonenumber_F_s">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#phonenumber_F_s{
background: #00bbf0;
border-radius:25px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="phonenumber_logo_s">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#phonenumber_logo_s{
background:rgb(0,0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/D:/grocery_store/grocery_store/assets/icons/phone-call.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="phonenumber_befor_s">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>219</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#phonenumber_befor_s{
background: rgb(0,0,0,0);
border-radius:25px;
color:#005792;
}</string>
    </property>
    <property name="text">
     <string>phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phonenumber_s">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>260</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kalameh Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#phonenumber_s{
background: #00bbf0;
border-radius:25px;
color:#00204a;
}</string>
    </property>
   </widget>
   <zorder>bottom_welcome</zorder>
   <zorder>Singup_label</zorder>
   <zorder>username_befor_s</zorder>
   <zorder>password_F_s</zorder>
   <zorder>password_logo_s</zorder>
   <zorder>password_befor_s</zorder>
   <zorder>password_s</zorder>
   <zorder>username_F_s</zorder>
   <zorder>user_logo_s</zorder>
   <zorder>username_s</zorder>
   <zorder>address_F_s</zorder>
   <zorder>address_logo_s</zorder>
   <zorder>address_befor_s</zorder>
   <zorder>address_s</zorder>
   <zorder>phonenumber_F_s</zorder>
   <zorder>phonenumber_logo_s</zorder>
   <zorder>phonenumber_befor_s</zorder>
   <zorder>phonenumber_s</zorder>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
